src/helm/benchmark/static/schema.yaml

```diff
@@ -671,22 +671,22 @@ models:
     todo: true
 
   # MosaicML
-  - name: together/mpt-7b
+  - name: huggingface/mpt-7b
```

Comment:
I wonder if the (hosting) organization should be `local` instead of `huggingface` since we are running things locally.

Reply:
We currently don't support [remote Hugging Face inference](https://huggingface.co/inference-api), so currently all Hugging Face models are run locally by default.

@julian-q's "local" workaround was to distinguish between models where the weights have to be manually downloaded to disk (i.e. LLaMA and Alpaca). Since mpt-7b's weights are public on Hugging Face Hub, we don't need this workaround here.

Reply:
For now, can we just call this `mosaicml/mpt-7b`?

Reply:
This is fine with me.  We should think through what the cache key is going to look like with creator, host, and implementation.  Nice thing about <org>/<model> is that it's simple.